### DISCUSSION TITLE
Are paths of `geoip.dat` and `geosite.dat` configurable?

### DISCUSSION THREAD

**Discussion by playgithub** [v2fly/v2ray-core#1720] [Q&A] [answered]
By config instead of overwriting, for it will not be overwritten by an upgrade.

**Comment by AkinoKaede** [comment, marked as answer]
V2Ray follows `XDG Base Directory Specification` (`master` branch), you could set the environment variables to change the path of assets.
<https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html>

Your can also set `v2ray.location.asset` or `V2RAY_LOCATION_ASSET` to change the path.

**Reply by playgithub** [reply]
Thanks
Anyway I prefer a config file
